app/src/main/java/io/homeassistant/companion/android/onboarding/manual/ManualSetupView.kt

```diff
@@ -79,3 +94,17 @@ fun ManualSetupView(
         }
     }
 }
+
+@Preview(showSystemUi = true)
```

Comment:
is this first annotation necessary here?

Reply:
@dshokouhi Yes, it shows one for dark mode and one for standard mode. Mostly just to compare the two at a glance when developing